### ISSUE TITLE
AA restarts frequently

### ISSUE THREAD

**Issue by nahueltaibo** [gartnera/headunit#180]
Starting about a week ago android auto started to close and restart. 
This happens when more while using maps and trying to play audio at the same time. 
Makes audio reproduction useless as it restarts AA every a couple seconds.

When only using maps it restarts every couple minutes.

I made no changes in my Mazda 3 2014 since the first install of this mod.

**Comment by QQrious** [comment]
I'm having the same problem with a Mazda 3 2016. At this point I think it is because of the new android update on the phone side.

**Comment by csirek** [comment]
Check this:
https://github.com/gartnera/headunit/issues/173

**Comment by nahueltaibo** [comment]
Thank you very much @csirek! I found the solution there!
The steps to solve this are:
Get the compressed update from [RobsonMi's solution](https://github.com/gartnera/headunit/issues/173#issuecomment-589630973) : 
https://github.com/gartnera/headunit/files/4235978/AndroidAuto_v1.05-52-g54b38d6.zip

1. Unzip AndroidAuto_v1.05-52-g54b38d6.zip
1. Copy contents of **installer** folder to USB stick
1. Plug the usb stick into your car
1. It should install automatically, (in my case took 3 or 4 minutes to show the prompt asking to start the installation) if it doesn't, restart navigation (nav button + mute for 10s) and follow the prompts.
1. Give 10 minutes to the installation process (It doesnt inform about the progress nor success)
1. After 10 minutes manually reboot CMI (nav btn + mute for 10s).

Those were the steps I followed and everything is working fine now.
